### PR TITLE
Added `--HEAD` support to cloc.

### PR DESCRIPTION
**Pull request by meskyanichi**

### CODE REVIEW

File: Library/Formula/cloc.rb [hunk 1]
```diff
@@ -3,11 +3,18 @@ class Cloc < Formula
   homepage "https://github.com/AlDanial/cloc/"
   url "https://github.com/AlDanial/cloc/releases/download/1.64/cloc-1.64.pl"
```
Comment (mikemcquaid) [inline Library/Formula/cloc.rb]: Use https://github.com/AlDanial/cloc/archive/1.64.tar.gz as the URL and then you can have the same logic for stable and head.

**Comment by meskyanichi** [comment]
Thanks for the tip @mikemcquaid! I've amended the commit.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `cloc: add HEAD.`.

**Comment by meskyanichi** [comment]
Gotcha. Thanks for the merge+feedback! :+1:

**Pull request closed without merging**